static/app/components/events/interfaces/spans/spanBar.tsx

```diff
@@ -118,11 +118,12 @@ type SpanBarProps = {
   toggleEmbeddedChildren:
     | ((props: {eventSlug: string; orgSlug: string}) => void)
     | undefined;
-  toggleSiblingSpanGroup: ((span: SpanType) => void) | undefined;
+  toggleSiblingSpanGroup: ((span: SpanType, occurrence?: number) => void) | undefined;
```

Comment:
I know you switched it, but I actually think it shouldn't be optional, conceptually, if you are picking a group, it must have occurred.

Reply:
Yeah true, I agree it shouldn't be optional at least for `spanGroupBar`, but I think it should be kept optional for `spanBar`, since this component is associated with a group only when it is part of an expanded sibling group. So regular `spanBars` won't have an occurrence.

Reply:
Nevermind you're right, I changed it so it's required here as well, since the function should always receive an occurrence number.